test/end-to-end/pages/screeners/starteligibility.js

```diff
@@ -3,16 +3,18 @@
 /* eslint-disable no-undef */
 const pageUnderTest = require('app/steps/ui/screeners/starteligibility');
```

Comment:
this hadn't been attended in previous performance implementation work, but perhaps it should, to avoid the pattern being replicated and used again; perhaps if this had been refactored the 'load the application object' pattern wouldn't have been reintroduced